src/test/run-cli-tests

```diff
@@ -30,7 +30,7 @@ if [ ! -e "$CRAM_BIN" ]; then
     # patched cram to support that. See upstream ticket at
     # https://bitbucket.org/brodie/cram/issue/9/allow-read-only-directories-for-t
     # -- tv@inktank.com
-    virtualenv "$VENV" && $VENV/bin/pip --log "$VENV"/log.txt install "$SRCDIR/downloads/cram-0.5.0ceph.2011-01-14.tar.gz"
+    virtualenv --python python3 "$VENV" && $VENV/bin/pip --log "$VENV"/log.txt install git+https://github.com/tchaikov/cram.git#egg=cram
```

Comment:
want to add a tag to the url? otherwise you can't push to the repo, without directly affecting the runs

Reply:
@sebastian-philipp thanks! now using "0.7-error-dir".